### PR TITLE
more strict

### PR DESCRIPTION
**Pull request by chrisnojima**

### CODE REVIEW

File: shared/actions/wallets.tsx [hunk 1]
```diff
@@ -452,12 +452,22 @@ const loadAccounts = async (
...
+    | WalletsGen.SelectAccountPayload,
+  msg: string,
+  err: RPCError
+) => {
   const errMsg = `Error ${msg}: ${err.desc}`
   // Assume that for auto-selected we're on the Wallets tab.
   if (
-    (action.type === WalletsGen.selectAccount && action.payload.reason === 'user-selected') ||
-    action.payload.reason === 'auto-selected'
+    action.type === WalletsGen.selectAccount &&
```
Comment (chrisnojima) [inline shared/actions/wallets.tsx]: @keybase/picnicsquad slightly logically different but i think what was intended

File: shared/actions/wallets.tsx [hunk 2]
```diff
@@ -525,21 +535,26 @@ const loadAssets = async (state: TypedState, action: LoadAssetsActions, logger:
   }
 }
 
-const createPaymentsReceived = (accountID, payments, pending, allowClearOldestUnread, error) =>
+const createPaymentsReceived = (
```
Comment (chrisnojima) [inline shared/actions/wallets.tsx]: @keybase/picnicsquad also changed

File: shared/actions/config/index.tsx [hunk 3]
```diff
@@ -562,17 +562,11 @@ const updateServerConfig = async (state: Container.TypedState, action: ConfigGen
...
-      features: {
-        admin?: {
-          value: boolean
-        }
-      }
-    } = JSON.parse(str.body)
-    const features = Object.keys(obj.features).reduce((map, key) => {
-      map[key] = obj.features[key as any]?.value ?? false
+    const obj: {features: any} = JSON.parse(str.body)
+    const features = Object.keys(obj.features).reduce<{[key: string]: boolean}>((map, key) => {
```
Comment (jakob223) [inline shared/actions/config/index.tsx]: Can you add a comment explaining what this is doing? I've read it through several times and still am not confident I have a good understanding.

Comment (chrisnojima) [reply shared/actions/config/index.tsx]: we're just getting a bag of key values out of some raw api call. the attempt at typing above doesn't really help us so just keep it as any. really this should be a real rpc so we get real types

**Pull request merged**